src/main/java/org/openstreetmap/atlas/checks/validation/intersections/LevelCrossingOnRailwayCheck.java

```diff
@@ -77,11 +78,11 @@ public class LevelCrossingOnRailwayCheck extends BaseCheck<Long>
             + "To fix: If the two ways should be on different layers then adjust the layer tags for each way appropriately. "
             + "If the two ways do intersect on the same layer then add the `railway=level_crossing` tag to this node.";
```

Comment:
As we discussed, lets make all of these instructions consistent with respect to the variables you added to replace 'railway=level_crossing'.